SwiftCompilerSources/Sources/SIL/Value.swift

```diff
@@ -113,6 +113,8 @@ extension Value {
 
   public var uses: UseList { UseList(bridged.getFirstUse()) }
   
+  public var firstUse: Operand? { Operand(bridged: bridged.getFirstUse()) }
```

Comment:
I can't see a use case for `firstUse`. The order of uses in not really predictable.